src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -115,11 +115,17 @@ public static double Abs(Complex value)
                 return double.PositiveInfinity;
             }
 
+            return Hypot(value._real, value._imaginary);
+        }
+
+        private static double Hypot (double a, double b)
```

Comment:
We'd really prefer to have submissions match the style guidelines, at least as much as is reasonable. I understand that it's annoying to push an update just to fix one character, but it's better than letting small inconsistencies build up over time. That said, I can go ahead and push a fix for this to your branch if you don't object to that.